docs/src/apps/jsonrpc-api.md

```diff
@@ -395,12 +395,39 @@ The signatures will be ordered based on the Slot in which they were confirmed in
 
 ```bash
 // Request
-curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0","id":1,"method":"getConfirmedSignaturesForAddress","params":["6H94zdiaYfRfPfKjYLjyr2VFBg6JHXygy84r3qhc3NsC", 0, 100]}' localhost:8899
+curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0","id":1,"method":"getConfirmedSignaturesForAddressDepricated","params":["6H94zdiaYfRfPfKjYLjyr2VFBg6JHXygy84r3qhc3NsC", 0, 100]}' localhost:8899
 
 // Result
 {"jsonrpc":"2.0","result":{["35YGay1Lwjwgxe9zaH6APSHbt9gYQUCtBWTNL3aVwVGn9xTFw2fgds7qK5AL29mP63A9j3rh8KpN1TgSR62XCaby","4bJdGN8Tt2kLWZ3Fa1dpwPSEkXWWTSszPSf1rRVsCwNjxbbUdwTeiWtmi8soA26YmwnKD4aAxNp8ci1Gjpdv4gsr","4LQ14a7BYY27578Uj8LPCaVhSdJGLn9DJqnUJHpy95FMqdKf9acAhUhecPQNjNUy6VoNFUbvwYkPociFSf87cWbG"]},"id":1}
 ```
 
+### getConfirmedSignaturesForAddress
+
+Returns confirmed signatures for transactions involving an
+address backwards in time from the provided slot
+
+#### Parameters:
+* `<string>` - account address as base-58 encoded string
+* `<object>` - (optional) Configuration object containing the following field:
+  * `slot: <u64>` - slot to start the search from.  If not provided then search from the highest max confirmed block
+  * `limit: <number>` - approximate maximum transaction signatures to return (between 1 and 100, default: 100).
```

Comment:
This is a pretty low cap. Maybe this is what you envision clients wanting, but from the blockstore/rpc perspective, I think there's no reason we couldn't support higher.

Reply:
Oops, I meant to also limit by max slots -- the lesser of `limit` transactions or 10,000 slots.     I'm thinking of the main consumers of this API (wallets, explorers) that only need to fetch enough history to allow the user to scroll for a screen or two while fetching the next chunk in the background.